Extension/.scripts/verify.ts

```diff
@@ -3,13 +3,50 @@
  * See 'LICENSE' in the project root for license information.
  * ------------------------------------------------------------------------------------------ */
 
-import { checkCompiled, checkPrep } from './common';
+import { checkBinaries, checkCompiled, checkDTS, checkPrep, error, green } from './common';
+const quiet = process.argv.includes('--quiet');
 
 export async function main() {
-    await checkPrep();
-    await checkCompiled();
+    let failing = await checkPrep() && (quiet || error(`Files are not up to date. Run ${green('yarn prep')} to fix it.`));
+    failing = (await checkCompiled() && (quiet || error(`Compiled files are not present. Run ${green('yarn compile')} to fix it.`))) || failing;
+    failing = (await checkBinaries() && (quiet || error(`The native binary files are not present. You should either build or install the native binaries\n\n.`))) || failing;
+    if (failing) {
+        process.exit(1);
+    }
+}
+
+export async function compiled() {
+    let failing = false;
+    failing = (await checkCompiled() && (quiet || error(`Compiled files are not present. Run ${green('yarn compile')} to fix it.`))) || failing;
+
+    if (failing){
```

Comment:
Add space between `)` and `{`. Is linter not working?

Reply:
There was no rule for that. Added one.